package.json

```diff
@@ -4,6 +4,10 @@
   "description": "100% typesafe dependency injection framework for TypeScript projects",
   "license": "MIT",
   "author": "Snap Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Snapchat/ts-inject.git"
+  },
```

Comment:
let's also add https://docs.npmjs.com/cli/v10/configuring-npm/package-json#homepage to point to https://snapchat.github.io/ts-inject/